### PR TITLE
nutanix ready & logo

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

For SUMA only, adds a line about being a nutanix partner, and includes the logo.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [ ] Uyuni

# Links

Fixes https://github.com/SUSE/spacewalk/issues/12735

### CODE REVIEW

**Comment by Loquacity** [comment]
SUMA PDF:

image

**Review by keichwa — APPROVED** [review]
I'm not that sure, but ok.

File: modules/client-configuration/pages/vhm-nutanix.adoc [hunk 1]
```diff
@@ -1,11 +1,20 @@
 [[virt-nutanix]]
 = Virtualization with Nutanix
 
-You can use Nutanix AHV virtual machines, with {productname} by setting up a virtual host manager (VHM).
+ifeval::[{suma-content} == true]
 
+{suse} is a Nutanix ecosystem partner.
```
Comment (paususe) [inline modules/client-configuration/pages/vhm-nutanix.adoc:6]:
```suggestion
{suse} is a Nutanix ecosystem partner and SUSE Manager has obtained the Nutanix-ready certification in the "Nutanix AHV integrated" category.

```

**Review by paususe — CHANGES_REQUESTED** [review]
Let's make the certification a bit more explicit.

**Review by paususe — APPROVED** [review]

**Pull request merged**